### PR TITLE
Daily report properties

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1026

### Change description ###
Charts for sending daily reports

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -10,6 +10,9 @@ java:
         - error-notifications-password
         - error-notifications-url
         - error-notifications-username
+        - reports-email-username
+        - reports-email-password
+        - reports-recipients
```
Comment (lgonczar) [inline charts/reform-scan-blob-router/values.yaml]: This one doesn't exist in bootstrap.yaml

File: charts/reform-scan-blob-router/values.aat.template.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 java:
   environment:
     CRIME_ENABLED: "true"
+    SEND_DAILY_REPORT_ENABLED: "false"
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.aat.template.yaml]: should be default in `values.yml` instead

File: charts/reform-scan-blob-router/values.preview.template.yaml [hunk 3]
```diff
@@ -45,6 +45,7 @@ java:
     BULK_SCAN_PROCESSOR_URL: http://bulk-scan-processor-aat.service.core-compute-aat.internal
     CRIME_DESTINATION_CONTAINER: bs-sit-scans-received
     CRIME_ENABLED: true
+    SEND_DAILY_REPORT_ENABLED: false
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.preview.template.yaml]: once in `values.yml` this is not needed either

File: charts/reform-scan-blob-router/values.yaml (hunk 1 above)
Comment (doncem) [inline charts/reform-scan-blob-router/values.yaml]: also missing in perftest and prod environments

Comment (doncem) [reply charts/reform-scan-blob-router/values.yaml]: bootstrap is not necessary when doing chart updates

**Comment by apereverzin** [comment]
PR-356

**Pull request closed without merging**